src/pages/en/guides/cms/contentful.md

```diff
@@ -0,0 +1,445 @@
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **A Contentful account and a Contentful space**. If you don't have an account, you can [sign up](https://www.contentful.com/sign-up/) for a free account and create a new Contentful space. You can also use an existing space if you have one. 
+
+    :::tip
+    If you have an empty Contentful space, check out [setting up a Contentful model](#setting-up-a-contentful-model) to learn how to create a basic blog model for your content.
+    :::
+
+3. **Contentful credentials** - Find the following credentials in **Settings > API keys**. If you don't have any API keys, create one by clicking in **Add API key**.
+
+    - **Contentful space ID** - The ID of your Contentful space. 
+    - **Contentful delivery access token** - The access token to consume _published_ content from your Contentful space.
+    - **Contentful preview access token** - The access token to consume _unpublished_ content from your Contentful space.
+
+### Setting up credentials
+
+To add your Contentful space's credentials to Astro, create an `.env` file in the root of your project with the following code:
+
+```ini title=".env"
+CONTENTFUL_SPACE_ID=YOUR_SPACE_ID
+CONTENTFUL_DELIVERY_TOKEN=YOUR_DELIVERY_TOKEN
+CONTENTFUL_PREVIEW_TOKEN=YOUR_PREVIEW_TOKEN
+```
+
+Now, you should be able to use these environment variables in your project. If you would like to have IntelliSense for your Contentful environment variables, you can create a `env.d.ts` file in the `src/` directory and configure `ImportMetaEnv` like this:
+
+```ts title="src/env.d.ts"
+interface ImportMetaEnv {
+  readonly CONTENTFUL_SPACE_ID: string;
+  readonly CONTENTFUL_DELIVERY_TOKEN: string;
+  readonly CONTENTFUL_PREVIEW_TOKEN: string;
+}
+```
+Check out [environment variables](/en/guides/environment-variables/) for more information on how to use them.
+
+Your root directory should now include these new files:
+
+```ini title="" ins={2-3}
+├── src/
+│   └── env.d.ts
+├── .env
+├── astro.config.mjs
+└── package.json
+```
+
+### Installing dependencies
+
+Install the following packages with your favorite package manager:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # install contentful and @contentful/rich-text-html-renderer with npm 
+  npm install contentful @contentful/rich-text-html-renderer
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # install contentful and @contentful/rich-text-html-renderer with pnpm
+  pnpm install contentful @contentful/rich-text-html-renderer
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # install contentful and @contentful/rich-text-html-renderer with yarn
+  yarn add contentful @contentful/rich-text-html-renderer
+  ```
+  </Fragment>
+</PackageManagerTabs>
```

Comment:
I think its very important to have the renderer in here since they are most likely going to need it. Maybe I can expand on what each package does. But at least for me, it took some time to find the right tool to parse rich text 😅.

Reply:
Okay gotcha. It makes sense to leave it here then, but I think we should describe what it is / how to use it so they're not wondering about it.
Maybe we change that ShopItem example so that the description is rich text and we use this renderer?